oauth2_http/java/com/google/auth/oauth2/UserCredentials.java

```diff
@@ -210,6 +196,33 @@ public AccessToken refreshAccessToken() throws IOException {
     return new AccessToken(accessToken, new Date(expiresAtMilliseconds));
   }
 
+  /**
+   * Returns a Google ID Token from the refresh token response.
+   *
+   * @param targetAudience the aud: field the IdToken should include. This is not unused for
```

Comment:
"This is not used", or "This can't be used" would make it more clear that is not a library choice but a limitation of the flow.

Reply:
updated